<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotAreaDock</class>
 <widget class="QWidget" name="PlotAreaDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>296</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="lName">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>23</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="KLineEdit" name="leName"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lComment">
         <property name="text">
          <string>Comment</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="KLineEdit" name="leComment">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lOpacity">
         <property name="text">
          <string>Opacity</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="sbOpacity">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>The opacity ranges from 0 to 100, where 0 is fully transparent and 100 is fully opaque.</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="lOpacityPercent">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>%</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>178</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Background</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lBackgroundType">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>23</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <widget class="KComboBox" name="cbBackgroundType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="lBackgroundFileName">
         <property name="text">
          <string>File name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="KLineEdit" name="kleBackgroundFileName">
         <property name="toolTip">
          <string>Specify the name of the image file.</string>
         </property>
         <property name="showClearButton" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="bOpen">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string> Select the file to import</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="lBackgroundColorStyle">
         <property name="text">
          <string>Style</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="KComboBox" name="cbBackgroundColorStyle"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lBackgroundImageStyle">
         <property name="text">
          <string>Style</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="KComboBox" name="cbBackgroundImageStyle"/>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="lBackgroundFirstColor">
         <property name="text">
          <string>First color</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="KColorButton" name="kcbBackgroundFirstColor"/>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="lBackgroundSecondColor">
         <property name="text">
          <string>Second color</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="KColorButton" name="kcbBackgroundSecondColor"/>
       </item>
       <item row="6" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>79</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Border</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="lBorderStyle">
         <property name="text">
          <string>Style</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>23</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="KComboBox" name="cbBorderStyle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lBorderColor">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="KColorButton" name="kcbBorderColor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lBorderWidth">
         <property name="text">
          <string>Width</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="sbBorderWidth">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>18</width>
           <height>118</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>KLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
